package.json

```diff
@@ -8,6 +8,7 @@
     "build": "gulp build",
     "build:electron": "electron-rebuild -w usb --useCache -s --debug",
     "check:all": "yarn prettier:check && yarn lint && yarn flow:test && yarn stylelint && yarn lockfile:check && yarn manage:translations",
+    "check:NixOS": "nix-build -A tests.runFlow && yarn prettier:check && yarn lint && yarn stylelint && yarn lockfile:check && yarn manage:translations",
```

Comment:
Couldn't see reference to this, is this used solely by CI?

Reply:
I added it just because in NixOS you cannot use flow 
![image](https://user-images.githubusercontent.com/4303099/150499788-9e297aa3-6071-4ff3-832e-b32127647591.png)

That is why I added an alternative to check:all 

But since we are moving to typescript I can remove this.